### PR TITLE
Manual form data & scraper.py fix

### PR DESCRIPTION
**Pull request by geramirez**
Updates
1.  added manual data for forms
2.  fixed scraper.py

fyi, the issue was that the "Notes:" node for some records doesn't have a strong tag. 

```
[<p>(877) 807-9215 (Fax)</p>, 
<p><strong>FOIA Requester Service Center:</strong> Phone: (202) 317-3475</p>, 
<p><strong>FOIA Public Liaison:</strong> Phone: (202) 317-3475</p>, 
<p><strong>Website: </strong><a href="http://www.irs.gov/uac/IRS-Freedom-of-Information">http://www.irs.gov/uac/IRS-Freedom-of-In formation</a></p>, 
<p><strong>Notes: </strong></p>, 
<p>This office has additional FOIA contact information that can be found by visiting its website.</p>]
```

Also, some keyword changes occurred when I updated the data.

### CODE REVIEW

**Comment by khandelwal** [comment]
In this pull request, how come we have request_form changes to the non-manual data? Where did those changes come from?

**Comment by geramirez** [comment]
@khandelwal they came from the [document](https://gist.github.com/ramirezg/295a50f89cb3dcd5a8ec) @rjmajma sent us last week. It contains the validated form data.

**Comment by konklone** [comment]
I had told @ramirezg this would be a good place to put it, @khandelwal. Do you think there's a better place?

(Just a note: anyone is free to review this, so I have unassigned myself. If I decide to review it later, I'll re-assign it to me.)

**Comment by khandelwal** [comment]
That's not quite the question I was asking. There are changes to contacts/data and contacts/manual_data in this pull request. I see why we have changes to manual_data. 

Why do we have changes to contacts/data?

File: contacts/manual_data/USAID.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+departments:
+- name: U.S. Agency for International Development
+  request_form: https://foiarequest.usaid.gov/index.aspx
```
Comment (khandelwal) [inline contacts/manual_data/USAID.yaml]: Does this link work for you? It doesn't for me.

File: contacts/manual_data/USDA.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+name: Department of Agriculture
+request_form: https://efoia-pal.usda.gov/palMain.aspx
```
Comment (khandelwal) [inline contacts/manual_data/USDA.yaml]: Does this link work for you? It doesn't for me.

File: contacts/manual_data/STB.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+departments:
+- name: Surface Transportation Board
+  request_form: http://199.79.178.189/stb/efilings.nsf/FOIARequest?OpenForm
```
Comment (khandelwal) [inline contacts/manual_data/STB.yaml:3]: Is this really the URL for this site? If I try to go to this from the GSA network, it's blocked.

**Comment by geramirez** [comment]
@khandelwal, oh got it. to test the manual data update I had to regenerate the data. For some reason the keywords tend to change whenever I update.

File: contacts/manual_data/STB.yaml (hunk 3 above)
Comment (geramirez) [inline contacts/manual_data/STB.yaml:3]: It's a horrible url but it works for me 
screen shot 2014-11-04 at 5 04 54 pm

File: contacts/manual_data/USDA.yaml (hunk 2 above)
Comment (geramirez) [inline contacts/manual_data/USDA.yaml]: Nope it's not working for me either, I wonder if it's temporarily or permanently down.

Comment (khandelwal) [inline contacts/manual_data/USDA.yaml]: @rjmajma Was this working when you added this in?

Comment (rjmajma) [inline contacts/manual_data/USDA.yaml]: It definitely was. Huh, that's strange.

Comment (konklone) [inline contacts/manual_data/USDA.yaml]: Both the USDA and USAID links use the same software, btw. You can see it by looking at their source code -- it's got JavaScript and HTML in there, it's just not showing any information.

**Comment by konklone** [comment]
Thanks for this, @ramirezg, this looks good to me now!

**Pull request merged**